docs/resources/team.md

```diff
@@ -24,6 +37,7 @@ The following arguments are supported:
 
 * `name` - (Required) The name of the team.
 * `description` - (Optional) A description for the team.
+* `memberships` - (Optional) A resource to tie a schedule or user to a team via a incident role.
```

Comment:
Do we also need to document the arguments that are valid for the `memberships` block?

Reply:
oh good callout, I had the example above but ya it doesn't reference the schedule_id